utils/formatSearchResult.js

```diff
@@ -80,6 +80,10 @@ function addPoste(elem, message) {
 }
 
 function addLinkJO(elem, message) {
+    if (elem.source_name == "JORF") {
+        message += `🔗 _Lien JO_:  [cliquez ici](https://www.legifrance.gouv.fr/jorf/id/${elem.source_id})\n`
+    }
+    return message
```

Comment:
This condition can be added in the `switch case` below (between line 91 & 92) which is meant to prevent "if forests" 😇
The code will look something like this:
```js
[...]
case 'BOMI':
        message += `🔗 _Lien JO_:  [cliquez ici](https://bodata.steinertriples.ch/${elem.source_id}.pdf)\n`
        break
case 'JORF':
        message += `🔗 _Lien JO_:  [cliquez ici](https://www.legifrance.gouv.fr/jorf/id/${elem.source_id})\n`
        break
default:
[...]
```